examples/new_workbench_project_from_geometry.py

```diff
@@ -0,0 +1,15 @@
+import flow360 as fl
+from flow360.component.geometry import Geometry
+
+fl.Env.dev.active()
+
+# geometry
+geometry_draft = Geometry.from_file(
+    "data/airplane_geometry.egads",
```

Comment:
don't point to any local file from examples. We want these scripts to be standalone so you can always copy-paste and execute. See other examples (e.g. here: https://github.com/flexcompute/Flow360/blob/develop/examples/run_case_from_files.py) for getting mesh file

Reply:
Based this example off `geometry_id_surface_volume_case_airplane_v1_from_egads.py` where the same thing was being done - I'll change it to the new format.

Reply:
Fixed in [6cece60](https://github.com/flexcompute/Flow360/pull/470/commits/6cece60781cc6aba8fc1ca2fdedd6b33711331be)